src/ok.rs

```diff
@@ -218,3 +229,43 @@ fn dev_sev_rw(file: &mut fs::OpenOptions) -> TestResult {
 
     TestResult { success, msg }
 }
+
+fn has_kvm_support() -> TestResult {
+    let mut success = true;
+    let path = "/dev/kvm";
+
+    let msg = match File::open(path) {
+        Ok(_) => match kvm_ioctls::Kvm::new() {
+            Ok(obj) => format!("KVM API version: {}", obj.get_api_version()),
+            Err(e) => {
+                success = false;
+                format!("Error obtaining KVM object: ({})", e.to_string().red())
```

Comment:
I'd suggest re-wording this to a more appropriate domain language. Maybe something like "accessing KVM device node failed"